matRad/gui/widgets/matRad_ViewingWidget.m

```diff
@@ -947,7 +951,7 @@ function UpdatePlot(this)
                 end
                 
                 %this creates a loop(needed the first time a dose cube is loaded)
-                if isempty(this.IsoDose_Levels) || ~this.NewIsoDoseFlag
+                if isempty(this.IsoDose_Levels) || ~this.NewIsoDoseFlag || ~this.checkUpdateNecessary({'viewer_options'},this.evt)
```

Comment:
I didnt entirely grasp the use for the NewIsoDoseFlag. so ive just left it in for now.